redis/src/cluster_async/mod.rs

```diff
@@ -454,12 +449,12 @@ where
         let inner = Arc::new(InnerCore {
             conn_lock: RwLock::new((connections, Default::default())),
             cluster_params,
+            pending_requests: Mutex::new(Vec::new()),
```

Comment:
why did you move it to InnerCore?

Reply:
otherwise it won't be accessible from within `execute_on_multiple_nodes`, and you won't be able to add the new requests to the queue.